### PR TITLE
transactions rpc methods

### PR DESCRIPTION
**Pull request by furkhat**
* global (not grouped) order of rpc methods docs
* returns slice explicitly without wrapper struct.

### CODE REVIEW

**Comment by furkhat** [comment]
If we decided that we don't need wrappers, I will adjust all other methods. @ababo @dzeckelev ?

**Comment by ababo** [comment]
It would be good.

File: doc/ui/rpc.md [hunk 1]
```diff
@@ -19,8 +19,6 @@ This document describes a UI JSON RPC API located in "ui" namespace.
 *Result (object)*:
 - `channel` (string) - id of channel to be created.
 
-### Settings
```
Comment (dzeckelev) [inline doc/ui/rpc.md]: Do not delete this. Please sort the methods according to the objects to which the methods relate.

Comment (ababo) [reply doc/ui/rpc.md]: @furkhat Why do you remove grouping? They seem to be reasonable, don't they?

File: doc/ui/rpc.md [hunk 2]
```diff
@@ -59,6 +57,20 @@ This document describes a UI JSON RPC API located in "ui" namespace.
...
+
+*Method*:	`createTemplate`
+
+*Description*: Create new template.
+
+*Parameters*:
+1. Password (string)
+2. Template (`data.Template` object)
+
+*Result (object)*:
```
Comment (ababo) [inline doc/ui/rpc.md]: Please remove the result wrappers as we agreed.

File: ui/transaction.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+	"strings"
+
+	"github.com/privatix/dappctrl/data"
+)
+
+// GetTransactions returns transactions by related object.
+func (h *Handler) GetTransactions(
+	password, relType, relID string) ([]data.EthTx, error) {
+	logger := h.logger.Add("method", "GetTransactions",
+		"relatedType", relType, "relatedID", relID)
```
Comment (ababo) [inline ui/transaction.go]: relatedId

File: doc/ui/rpc.md [hunk 4]
```diff
@@ -16,7 +17,7 @@ This document describes a UI JSON RPC API located in "ui" namespace.
 1. Password (string)
 2. Account id (string)
 
-*Result (array of bytes)*: private key.
+*Result (array)*: private key as array of bytes.
```
Comment (ababo) [inline doc/ui/rpc.md]: I don't really like this approach, I think we need to provide an exact type within round brackets, and afterwards the description.

File: doc/ui/rpc.md [hunk 5]
```diff
@@ -39,7 +40,7 @@ This document describes a UI JSON RPC API located in "ui" namespace.
 *Parameters*:
 1. Password (string)
 
-*Result (array of `data.Account` objects)*: account objects.
+*Result (array)*: of `data.Account` objects.
```
Comment (ababo) [inline doc/ui/rpc.md]: The same here.

File: doc/ui/rpc.md [hunk 6]
```diff
@@ -94,6 +95,51 @@ This document describes a UI JSON RPC API located in "ui" namespace.
 
 *Result*: None.
 
+
+### Auth
```
Comment (ababo) [inline doc/ui/rpc.md]: Authentication

File: doc/ui/rpc.md [hunk 7]
```diff
@@ -94,6 +95,51 @@ This document describes a UI JSON RPC API located in "ui" namespace.
...
+
+#### Set password
+
+*Method*: `setPassword`
+
+*Description*: Sets the password. Meant to be called only once. For password updates use `updatePassword`.
```
Comment (ababo) [inline doc/ui/rpc.md]: To update password use ...

File: doc/ui/rpc.md [hunk 8]
```diff
@@ -211,50 +245,38 @@ This document describes a UI JSON RPC API located in "ui" namespace.
...
 
 
-### Objects
+### Transactions
 
-#### Get object
+#### Get Transactions
 
-*Method*:	`getObject`
+*Method*:	`getTransactions`
```
Comment (ababo) [inline doc/ui/rpc.md]: Maybe `getEthTransactions`?

File: ui/transaction.go [hunk 9]
```diff
@@ -0,0 +1,50 @@
+package ui
```
Comment (ababo) [inline ui/transaction.go]: Maybe to rename into `eth_tx.go`?

File: doc/ui/rpc.md [hunk 10]
```diff
@@ -211,50 +245,38 @@ This document describes a UI JSON RPC API located in "ui" namespace.
...
+### Objects
 
-#### Get Templates
+#### Get object
 
-*Method*:	`getTemplates`
+*Method*:	`getObject`
 
-*Description*: Get templates.
+*Description*: Get an object of a specified type..
```
Comment (ababo) [inline doc/ui/rpc.md]: Doubled period.

File: doc/ui/rpc.md [hunk 11]
```diff
@@ -211,50 +245,38 @@ This document describes a UI JSON RPC API located in "ui" namespace.
...
+### Transactions
 
-#### Get object
+#### Get Transactions
 
-*Method*:	`getObject`
+*Method*:	`getTransactions`
 
-*Description*: Get an object of a specified type..
+*Description*: Get transactiosn.
```
Comment (ababo) [inline doc/ui/rpc.md]: Get Ethereum transaction.

File: ui/transaction.go (hunk 3 above)
Comment (ababo) [reply ui/transaction.go]: You didn't fix that as well.

**Pull request merged**